components/CreateReplyRequestForm/CreateReplyRequestForm.js

```diff
@@ -0,0 +1,238 @@
+  button: {
+    backgroundColor: theme.palette.primary[500],
+    color: theme.palette.common.white,
+    cursor: 'pointer',
+    border: 'none',
+    outline: 'none',
+    borderRadius: 30,
+    padding: '10px 13px',
+  },
+  buttonGroup: {
+    flex: 3,
+    paddingLeft: 8,
+    display: 'flex',
+    alignItems: 'center',
+    whiteSpace: 'nowrap',
+    '& button': {
+      flex: 1,
+      marginRight: 0,
+      color: theme.palette.secondary[300],
+      background: theme.palette.common.white,
+      outline: 'none',
+      cursor: 'pointer',
+      fontSize: 16,
+      padding: '8px 24px',
+      '&:first-child': {
+        borderTopLeftRadius: 30,
+        borderBottomLeftRadius: 30,
+        paddingLeft: 24,
+      },
+      '&:last-child': {
+        borderTopRightRadius: 30,
+        borderBottomRightRadius: 30,
+        paddingRight: 24,
+      },
+      '&:not(:first-child):not(:last-child)': {
+        borderLeft: 0,
+        borderRight: 0,
+      },
+      '&:hover': {
+        background: theme.palette.secondary[50],
+      },
+      '&.active': {
+        color: theme.palette.primary[500],
+      },
+    },
+  },
+  form: {
+    position: 'relative',
+    flex: 1,
+  },
+  textarea: {
+    borderRadius: 24,
+    width: '100%',
+    border: `1px solid ${theme.palette.secondary[100]}`,
+    padding: '17px 14px',
+    '&:focus': {
+      paddingBottom: 17 + 37,
+      outline: 'none',
+    },
+  },
+  submit: {
+    position: 'absolute',
+    bottom: 12,
+    right: 8,
+  },
+  replyButton: {
+    flex: 1,
+    width: '100%',
+  },
+  menu: {
+    marginTop: 40,
+    marginLeft: 30,
+  },
+}));
+
+const CREATE_REPLY_REQUEST = gql`
+  mutation CreateReplyRequestFromForm($articleId: String!, $reason: String!) {
+    CreateReplyRequest(articleId: $articleId, reason: $reason) {
+      id
+    }
+  }
+`;
+const MIN_REASON_LENGTH = 80;
+
+const SubmitButton = ({ className, articleId, text, disabled, onFinish }) => {
+  const [createReplyRequest] = useMutation(CREATE_REPLY_REQUEST, {
+    refetchQueries: ['LoadArticlePage'],
+  });
+
+  const handleSubmit = e => {
+    e.preventDefault(); // prevent reload
+    if (disabled) return;
+    createReplyRequest({ variables: { articleId, reason: text } });
+    onFinish();
+  };
+
+  return (
+    <button className={className} onClick={handleSubmit} disabled={disabled}>
+      {disabled ? '字數太少，無法送出' : '送出理由'}
+    </button>
+  );
+};
+
+const CreateReplyRequestForm = React.memo(
+  ({ articleId, requestedForReply, onNewReplyButtonClick }) => {
+    const [disabled, setDisabled] = useState(false);
+    const [showForm, setShowForm] = useState(false);
+    const [text, setText] = useState('');
+
+    const [shareAnchor, setShareAnchor] = useState(null);
+
+    useEffect(() => {
+      // restore from localStorage if applicable.
+      // We don't do this in constructor to avoid server/client render mismatch.
+      //
+      setText(localStorage.text || text);
```

Comment:
Actually this collides with reply's localstorage cache. This is a bug that can be perceived in current production as well. Changing `text` to some other key should fix the bug; we can also open a separate issue, since this is not in the scope of this PR actually.

![螢幕快照 2020-05-16 下午6 53 59](https://user-images.githubusercontent.com/108608/82118015-e88ad780-97a6-11ea-8541-057b3cb7ed06.png)

Reply:
card added here
https://github.com/orgs/cofacts/projects/5#card-38799032